src/modules/github/api.ts

```diff
@@ -9,5 +9,6 @@ export interface SearchRepositoriesResponse {
     items: GithubRepositoryItem[];
 }
 
-export const searchRepositories = (name: string) => api.get<SearchRepositoriesResponse>(`search/repositories?q=${name}`);
+export const searchRepositories = (name: string, perPage: number, currentPage: number) =>
```

Comment:
Simplify argument names:
```suggestion
export const searchRepositories = (name: string, size: number, page: number) =>
```